engines/group/spec/controllers/groups_controller_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
-require_relative '../../../support/shared_examples/controllers.rb'
-require_relative '../../../support/authentication.rb'
+require 'support/shared_examples/controllers.rb'
+require 'support/authentication.rb'
 
-describe Api::V1::GroupsController do
+describe Group::Api::V1::GroupsController do
```

Comment:
No idea. Any help would be welcome, cause I couldn't manage to make it work. Looks like you moved the code but not the specs. I'm still figuring out how it should be.